libcflib/whoosh/writing.py

```diff
@@ -29,7 +31,7 @@ def get_value(fields, fieldname):
     return value
 
 
-def get_fieldnames(fields, base=""):
+def get_fieldnames(fields, base=[]):
```

Comment:
You should not have mutable values as a default argument.  Pleas change this to an empty tuple or None, if needed.